nixos/modules/services/display-managers/lemurs.nix

```
@@ -0,0 +1,118 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+let
+  cfg = config.services.displayManager.lemurs;
+  settingsFormat = pkgs.formats.toml { };
+in
+{
+  options.services.displayManager.lemurs = {
+    enable = lib.mkEnableOption "lemurs, a customizable TUI display/login manager";
```

Should probably add a note about the seat issue?

```suggestion
    enable = lib.mkEnableOption "" // {
      description = ''
        Whether to enable lemurs, a customizable TUI display/login manager.

        Note: For Wayland compositors, your user must be in the "seat" group.
      '';
    };
```